src/components/Header.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import logo from '../redditlogo.svg';
+
+const StyledHeader = styled.header`
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 32px;
+`;
+const StyledNav = styled.nav`
```

Comment:
It's common to add a line break between the styled components. Just makes it a bit easier to read

Reply:
Gotcha 👍  I'll make those changes.